### DISCUSSION TITLE
Discord integration for deployment notifications

### DISCUSSION THREAD

**Discussion by michavie** [vercel/community#69] [Idea]
discord has gained a lot of popularity in the last years and has become a de-facto standard for many internet communities & organizations

thus, consider this a request for a dedicated discord integration — similar to the slack one — for deployment notification and similar.

**Comment by MathiasReker** [comment]
Discord has an API. Have you tried that?

**Comment by bonesoul** [comment]
+1 on this

**Comment by brianespinosa** [comment]
It is probably worth noting that there is an [official integration for Slack listed under "Messaging"](https://vercel.com/integrations#messaging) and seeing that Vercel even has a Discord server for Next.js it is a little surprising that there is no official Discord integration that does the exact same thing.

Sure there is an Integration API but I am unlikely to try to create yet another half-baked integration that might be superseded by something officially supported. There are also other very incomplete Discord integrations others have tried to create using the most semantically named urls for the Integrations system. I am not interested in creating something under `https://vercel.com/integrations/discord-bot-three`. I certainly would not give authorization to that bot to have access to my Vercel projects.

**Comment by drnachio** [comment]
Please. we need it!

**Comment by FrancoAguzzi** [comment]
Pleaseeeee, we need it²!

**Comment by rohanmendon** [comment]
please, we need it

**Comment by r4kh4t** [comment]
Also requesting Vercel to support this

**Comment by ItsNiceCraft** [comment]
+1

**Comment by raflymln** [comment]
+1

**Comment by vycos-zen** [comment]
+1 

I would love to have an integration to Discord. It has become a pretty robust platform. All the events to be propagated to a channel can really be a big help.

**Comment by JonathanAmdahl** [comment]
+1 Please add this 🙏

**Comment by mengxi-ream** [comment]
+1

**Comment by gemblue** [comment]
Only Slack was supported officially

I am waiting for Discord 🙏

**Reply by OreczXOfficial** [reply]
I'm creating and endpoint (API) in my domain try: bedrockapi.mubi.tech/api/webhook/vercel?webhook=your webhook here

**Reply by OreczXOfficial** [reply]
Okay stop using "?" Do bedrockapi.mubi.tech/api/webhook/vercel/Webhook here
